projects/imagenet/local_run.py

```diff
@@ -35,9 +35,9 @@
 import torch.multiprocessing as multiprocessing
 
 from experiments import CONFIGS
-from nupic.research.frameworks import vernon
 from nupic.research.frameworks.vernon import ImagenetExperiment
 from nupic.research.frameworks.vernon.parser_utils import MAIN_PARSER, process_args
+from nupic.research.frameworks.vernon.run import run, terminate_processes
```

Comment:
Maybe we can look for a better name later so we don't have a `from run import run` situation. Although it is quite common in python like `time.time` or `copy.copy`.

Reply:
Good point. That's why I had it the other way before. There's a trick to make `run` a callable module, but I figured this would be simpler for now.

Reply:
@mvacaporale You should remove the `import ray` and the `FIXME` from this file and try to run any experiment.
https://github.com/numenta/nupic.research/pull/355/files#diff-87f6f72f74eee2681c045d1824029eafR33-R34